src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,76 @@
+package duke.tasks;
+
+import duke.DukeException;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+public class TaskList {
+    private ArrayList<Task> l = new ArrayList<>();
```

Comment:
perhaps it will be better to name this variable in a more meaningful way , for example, tasks